packages/app/src/utils/is-absolute-url.ts

```diff
@@ -1 +1,3 @@
-export const isAbsoluteUrl = (url: string) => /^https?:\/\//.test(url);
+export function isAbsoluteUrl(url: string) {
+  return /^https?:\/\//.test(url);
```

Comment:
I just realize a url starting with `//` is also an absolute url
```suggestion
  return /^(https?:)?\/\//.test(url);
```